spec/features/invite_user_sign_up_spec.rb

```diff
@@ -43,11 +47,16 @@ def accept_invitation(user)
     current_email.click_link "Accept this invitation"
     fill_in "Password", with: user.password
     fill_in "Full Name", with: user.full_name
+    fill_in "Credit Card Number", with: "4242424242424242"
+    fill_in "Security Code", with: "123"
+    select "7 - July", from: "date_month"
+    select 1.year.from_now.year, from: "date_year"
     click_button "Sign Up"
+    expect(page).to have_css('.alert-success')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.